cm/ut/cm.c

```diff
@@ -47,10 +46,9 @@ char  *cm_ut_server_args[] = { "m0d", "-T", "LINUX",
 
 static void cm_ut_server_start(void)
 {
-	int rc;
-
-	cm_ut_sctx.rsx_xprts         = &xprt;
-	cm_ut_sctx.rsx_xprts_nr      = 1;
+	int                 rc;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated